src/index.js

```diff
@@ -20,10 +20,12 @@ const CHILD_SEED_TYPES = {
   SPAWN: 'spawn',
   VOTING: 'voting',
   MANAGEMENT: 'management',
-  NETWORK: 'network'
+  NETWORK: 'network',
+  BITCOIN: 'bitcoin'
 }
 
 const DERIVATION_PATH = "m/44'/60'/0'/0/0"
+const BTC_DERIVATION_PATH = "m/44'/0'/0'/0/0"
```

Comment:
Is there are reason for using 44 instead of 84 addresses (segwit/Bech32)?

Reply:
There isn't. Good catch.